src/app/stores/action_handlers/document.ts

```diff
@@ -106,14 +108,7 @@ export default function(REG: ActionHandlerRegistry<AppStore, Actions.Action>) {
                 const { chart } = instance;
 
                 for (const property of template.properties) {
-                  if (property.target.property) {
-                    CharticulatorContainer.ChartTemplate.SetChartProperty(
-                      chart,
-                      property.objectID,
-                      property.target.property,
-                      property.default
-                    );
-                  } else {
+                  if (property.target.attribute) {
```

Comment:
You removed the `SetChartProperty` logic here completely, was that intentional?

Reply:
Yes